src/Modules/AuthModule/components/ForgetPass/ForgetPass.tsx

```diff
@@ -1,5 +1,95 @@
-import React from "react";
+import { useForm } from "react-hook-form";
+import logo from "../../../../assets/images/PMS 3.png";
+import { useNavigate } from "react-router-dom";
+import { useState } from "react";
+import axios from "axios";
+import { useToast } from "../../../../Context/ToastContext";
+import { FormData } from "../../../../interfaces/Auth";
 
 export default function ForgetPass() {
-  return <>ForgetPass</>;
+	const [spinner, setSpinner] = useState<boolean>(false);
```

Comment:
When providing a default value for the hook, the type is automatically inferred from it. We don't need to set the type manually here, so we can remove `<boolean>`